components/director/pkg/graphql/app_template_validation_test.go

```diff
@@ -265,8 +265,7 @@ func TestApplicationTemplateInput_Validate_AccessLevel(t *testing.T) {
 func TestApplicationTemplateInput_Validate_Webhooks(t *testing.T) {
 	webhookInput := fixValidWebhookInput(inputvalidationtest.ValidURL)
 	invalidWebhookInput := fixValidWebhookInput(inputvalidationtest.ValidURL)
-	invalidWebhookInput.URL = nil
-
+	invalidWebhookInput.OutputTemplate = stringPtr(`{ "gone_status_code": 404, "success_status_code": 200}`)
```

Comment:
Instead of changing the test case, add another "invalid" one:
```suggestion
	invalidWebhookOutputTmpl := fixValidWebhookInput(inputvalidationtest.ValidURL)
	invalidWebhookOutputTmpl.OutputTemplate = stringPtr(`{ "gone_status_code": 404, "success_status_code": 200}`)

	invalidWebhookURL := fixValidWebhookInput(inputvalidationtest.ValidURL)
	invalidWebhookURL.URL = nil
```
Make sure to include what is "invalid" in the test case message